src/energyplus/ForwardTranslator/ForwardTranslateWaterHeaterMixed.cpp

```diff
@@ -58,6 +58,12 @@ boost::optional<IdfObject> ForwardTranslator::translateWaterHeaterMixed( WaterHe
   boost::optional<double> value;
   boost::optional<Schedule> schedule;
 
+  if (!modelObject.plantLoop() && !(modelObject.peakUseFlowRate() && modelObject.useFlowRateFractionSchedule()) ) {
+    LOG(Warn, modelObject.briefDescription() << " will not be translated as it not on a PlantLoop, and it does not have both a Peak Use Flow Rate "
+        "and a Use Flow Rate Fraction Schedule which is a required condition for stand-alone operation");
+    return boost::none;
+  }
```

Comment:
BUT do not actually translate them to IDF if either:
* On a PlantLoop
* has both the peakUseFlowRate set (optional double) **and the** useFlowRateFractionSchedule (optional schedule).


Technically I think we could do without the useFlowRateFractionSchedule but whatever, **it is desired here I think**, as it'll avoid people setting the flow rate by mistake (in OS App for eg...) and ending up with "hidden" standalone heaters if they get orphaned from a plantloop.

E+ source shows that no severe will be issued if schedule not specified while standalone operation, and it basically amounts to using an Unity schedule:

https://github.com/NREL/EnergyPlus/blob/68c51b98f26f703a0f7b92ac7c30a18f8a526313/src/EnergyPlus/WaterThermalTanks.cc#L2577-L2586

https://github.com/NREL/EnergyPlus/blob/68c51b98f26f703a0f7b92ac7c30a18f8a526313/src/EnergyPlus/WaterThermalTanks.cc#L5969-L5987


@kbenne